scripts/imagebuild.sh

```diff
@@ -54,6 +54,8 @@ echo "Pushing image with tag '${TAG}'."
 docker push ${REGISTRY}${REPO}/federation-v2:${TAG}
 
 if ! [ -z "${LATEST}" ]; then
```

Comment:
since `nounsent` is set, I think you're going to want to say `${LATEST:""}` here